PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/resources/META-INF/ruleConfigs.json

```diff
@@ -117,5 +121,13 @@
     ],
     "servicesToCheck": "KeyCredential",
     "antiPatternMessage": "Endpoint should not be used with KeyCredential for non-Azure OpenAI clients"
+  },
+  "UseOfBlockOnAsyncClientsCheck": {
+    "methodsToCheck": "block",
```

Comment:
We should check for other methods too - blockFirst, blockLast, toIterable, toStream etc.

Reply:
added: "block",
      "blockOptional",
      "blockFirst",
      "blockLast",
      "toIterable",
      "toStream",
      "toFuture",
      "blockFirstOptional",
      "blockLastOptional"